openapi/resource-server.yaml

```diff
@@ -690,6 +700,20 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           description: Incoming Payment Not Found
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              additionalProperties: false
+              properties:
+                walletAddress:
+                  type: string
+                  description: The URL of the wallet address that should receive the incoming payment.
+            examples:
+              Complete incoming payment for https://openpayments.guide/alice:
+                value:
+                  walletAddress: 'https://openpayments.guide/alice/'
```

Comment:
maybe this can just be the `ilp.money` wallet address one will be good so it works with Blair's PR regardless who merges first